packages/auth/amplify_auth_cognito/lib/src/flows/hosted_ui/hosted_ui_platform_flutter.dart

```diff
@@ -95,15 +95,18 @@ class HostedUiPlatformImpl extends io.HostedUiPlatformImpl {
   }
 
   @override
-  Future<void> signOut() async {
+  Future<void> signOut({
+    required CognitoSignInWithWebUIOptions options,
```

Comment:
Should we create a `SignOutOptions` class, even if it is just an alias for `CognitoSignInWithWebUIOptions` right now? If we ever have a need to add non hosted ui options, we would need to change this signature.

Reply:
Sounds good to me! I'll do it in a follow up if that works for you since rebasing on this branch I'm pretty sure would summon Cthulu from the 5th dimension

Reply:
lol. Yeah thats fine

Reply:
Had to merge this manually anyway so managed to sneak it in